src/App.tsx

```diff
@@ -1,61 +1,128 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { ToDo } from './components/TSTypes/ToDo';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const todosWithUsers = todosFromServer.map(todo => {
+  const user = usersFromServer.find(usr => usr.id === todo.userId);
+
+  return {
+    ...todo,
+    user,
+  };
+});
+
+export const App: React.FC = () => {
+  const [toDoList, setToDoList] = useState<ToDo[]>(todosWithUsers);
+
+  const [title, setTitle] = useState<string>('');
+  const [userId, setUserId] = useState<number>(0);
+
+  const [titleError, setTitleError] = useState<boolean>(false);
+  const [userError, setUserError] = useState<boolean>(false);
+
+  const resetFields = () => {
+    setTitle('');
+    setTitleError(false);
+
+    setUserId(0);
+    setUserError(false);
+  };
+
+  const formSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
The function name must begin with a verb:
```suggestion
  const submitForm = (event: React.FormEvent<HTMLFormElement>) => {
```